app/models/member.rb

```diff
@@ -86,4 +88,10 @@ def token_payload
       authentication_id: authentication.try(:id)
     }
   end
+
+  private
+
+  def existing_memberships_and_email_changed?
+    Member.where(email: email).where.not(id: id).any? && email_changed?
```

Comment:
@osahyoun quoting you: `Validate uniqueness if email has changed and other records exists with the same email address.`

`and other records exist with the same email address` => that's uniqueness. We're using rails uniqueness validator only if the email is already unique. It means you're skipping the uniqueness validator if the email is not unique.